vaapi/vaapidisplay_unittest.cpp

```diff
@@ -101,19 +101,6 @@ class VaapiDisplayTest : public ::testing::Test {
 #define VAAPI_DISPLAY_TEST(name) \
     TEST_F(VaapiDisplayTest, name)
 
-#if __ENABLE_X11__
-VAAPI_DISPLAY_TEST(CreateInvalidHandleX11) {
-    FAIL(); // remove when fixed;
-
-    // FIXME: This causes segmentation fault.
-    // VaapiDisplay should be able to handle this gracefully
-    NativeDisplay native = {1, NATIVE_DISPLAY_X11};
-    DisplayPtr display = VaapiDisplay::create(native);
-
-    EXPECT_FALSE(display.get());
-}
-#endif
-
```

Comment:
It doesn't matter where it crashes.  The point is we are testing that invalid input to our library function produces the "expected" outcome... in this case, we expect it to crash to tell the library user that they are doing something very wrong.  And a test such as this not only exercises our "negative" code path, it documents it too.